plugins/importexport/crossref/filter/ArticleCrossrefXmlFilter.inc.php

```diff
@@ -90,7 +90,7 @@ function createJournalArticleNode($doc, $submission) {
 
 		$journalArticleNode = $doc->createElementNS($deployment->getNamespace(), 'journal_article');
 		$journalArticleNode->setAttribute('publication_type', 'full_text');
-		$journalArticleNode->setAttribute('metadata_distribution_opts', 'any');
+// DEPRECATED in current scheme 4.4.2:	$journalArticleNode->setAttribute('metadata_distribution_opts', 'any');
```

Comment:
@klausru, could you include this in your other PR (https://github.com/pkp/ojs/pull/2668) to upgrade the schema version? Feel free to just remove the line there entirely.